src/components/theme/View/View.jsx

```diff
@@ -229,6 +230,27 @@ class View extends Component {
           )}
           <title>{this.props.content.title}</title>
           <meta name="description" content={this.props.content.description} />
+
+          <meta property="og:title" content={this.props.content.title} />
```

Comment:
Isn't that something that the editor would like to override? I would guess that if you embed a page without this tag, that there is a fallback for the site title anyways (haven't checked).

Reply:
AFAIK we currently don't have access to the site title